cli/lib/kontena/cli/stacks/yaml/opto/prompt_resolver.rb

```diff
@@ -51,7 +51,7 @@ def bool
       end
 
       def ask
-        prompt.ask(question_text, default: option.default)
+        prompt.ask(question_text, default: option.default, echo: option.handler.options[:echo].nil? ? true : option.handler.options[:echo])
```

Comment:
This seems a bit too magical (and long) line to me. Is it possible that `option.handler.options` is nil?